File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        res = 0
        curr = 0

        for i in nums:
            if i: curr += 1
            else:
                res = max(curr, res)
                curr = 0

        res = max(curr, res)
        return res

File: Data Structures & Algorithms/concatenation-of-array/submission-0.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> res(nums.size()<<1);
        for (int i = 0; i < nums.size(); i++) {
            res[i] = nums[i];
            res[i+nums.size()] = nums[i];
        }

        return res;
    }
};